packages/client/src/services/credit/creditRfqRequests.ts

```diff
@@ -43,6 +53,24 @@ export const createCreditRfq$ = (request: CreateRfqRequest) => {
   )
 }
 
+export const rfqRequestConfirmation$ = createdCreditRfq$.pipe(
```

Comment:
I was going to say, we must have had this set up before for the banner/pill notification .. that's [here](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/2286/files#diff-8f97e13b4d29ee18eff60a545dc7079aadd9a0bd3f6add44fafa529adf81f8c7R75)
.. I guess that whole file is going away now

Reply:
makes much more sense in the services area, btw 👍🏻